Sources/Sentry/include/SentryHttpTransport.h

```diff
@@ -8,15 +8,18 @@
 #import "SentryRequestManager.h"
 #import "SentryRateLimits.h"
 
+@class SentryEnvelopeRateLimit;
+
 NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryHttpTransport : NSObject <SentryTransport>
 SENTRY_NO_INIT
 
 - (id)initWithOptions:(SentryOptions *)options
     sentryFileManager:(SentryFileManager *)sentryFileManager
- sentryRequestManager:(id<SentryRequestManager>) sentryRequestManager
-     sentryRateLimits:(id<SentryRateLimits>) sentryRateLimits;
+ sentryRequestManager:(id<SentryRequestManager>)sentryRequestManager
+     sentryRateLimits:(id<SentryRateLimits>)sentryRateLimits
```

Comment:
l: to be honest I'd bundle these types (`SentryEnvelopeRateLimit` and `SentryRateLimits`) in a single one to avoid type explosion in the SDK.